src/main/scala/scynamo/wrapper/YearMonthFormatter.scala

```diff
@@ -0,0 +1,11 @@
+package scynamo.wrapper
+
+import java.time.format.DateTimeFormatter
+
+/** This is a custom formatter for `YearMonth` because
+  * "Years outside the range 0000 to 9999 must be prefixed by the plus or minus symbol."
+  * but the plus symbol is not added by the default `.toString`.
+  */
+object YearMonthFormatter {
```

Comment:
Should we make this `private[scynamo]` to be on the safe side?

Reply:
Made it `private[scynamo]` in https://github.com/moia-oss/scynamo/pull/255/commits/4ce09fa8460a60ed2ded648b4c6b4603e6466c4d.